src/txmempool.cpp

```diff
@@ -155,7 +125,9 @@ void CTxMemPool::UpdateForDescendants(txiter updateIt, cacheMap& cachedDescendan
             }
         }
     }
-    mapTx.modify(updateIt, update_descendant_state(modifySize, modifyFee, modifyCount));
+    mapTx.modify(updateIt, [modifySize, modifyFee, modifyCount](CTxMemPoolEntry& e) {
```

Comment:
```suggestion
    mapTx.modify(updateIt, [&/=](CTxMemPoolEntry& e) {
```

Is there any downside to just taking a reference/copy of everything that is used inside the lambda? This would make the code not too much more verbose than it previously was, and it would still be clear that every symbol used in the lambda is captured by reference/copy.

Reply:
Good point, no downside. Changed them all to default-capture by value since they are pointers and int arguments.